src/main/java/com/hazelcast/aws/AwsDiscoveryStrategy.java

```diff
@@ -63,6 +64,9 @@
     private static final int DEFAULT_CONNECTION_TIMEOUT_SECONDS = 10;
     private static final int DEFAULT_READ_TIMEOUT_SECONDS = 10;
 
+    // TODO: To be replaced with PartitionGroupMetaData.PARTITION_GROUP_PLACEMENT
```

Comment:
I'm not sure we want to replace it since it would break the backward-compatibility. You can either remove this comment or add that we can update it with the major release.